sites/common/includes/base/tab-navigation-bar.html

```diff
@@ -18,10 +17,10 @@
     // Determine which link this is and update.
     if (linkHost.includes('thunderbird.net')) {
```

Comment:
With the `data-link-id` you can query link.dataset['link-id'] (or something to that effect) instead of relying on a partial url match.